parameters.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
Herre it would be nice to create a comment of wat the script does, whos the author
Smth like this, https://bashcommands.com/bash-script-header there are many formats

+ The script probably should not be called "parameters.sh"